reana_job_controller/htcondorcern_job_manager.py

```diff
@@ -90,11 +89,15 @@ def __init__(
         self.workflow = self._get_workflow()
         self.unpacked_img = unpacked_img
 
+        # We need to import the htcondor package later during runtime after the Kerberos environment is fully initialised.
+        # Without a valid Kerberos ticket, importing will exit with "ERROR: Unauthorized 401 - do you have authentication tokens? Error "/usr/bin/myschedd.sh |"
+        initialize_krb5_token(workflow_uuid=self.workflow_uuid)
```

Comment:
- Can you please add yourself to the `AUTHORS.rst` file?

- Can you add a note to the `CHANGES.rst` file about this change? For example: "Amends CERN HTCondor compute backend to use the new myschedd connection library"

Reply:
Done!